### ISSUE TITLE
Relicensing

### ISSUE THREAD

**Issue by necauqua** [necauqua/chiseled-me#84]
Hey, I want to change the license from Apache 2.0 (the current one) to MIT (which it should've been from the beginning anyway) and this means that I need explicit agreement from the contributors, who are apparently equal owners of this software.

Technically I am the only owner of the repo with write access, but all the contributors still 'own' this exact piece of software (so I can't, say, sell it or relicense it without your argeement as it contains your changes).
If anyone wants to actully modify this code without my/our permission, they can create a fork where they will have to include the license notice file (with my name on top of it) anyway.

Basically from my own understanding, MIT is pretty much the same as Apache, only lacking like a feature or two that I/we will never use anyway, it's a Minecraft mod after all, c'mon, and way, way shorter, I like it a lot and use in most of my open source projects.

- [x] @MUTEDGALAXY
- [x] @Vexatos
- [x] @3TUSK 

^ I need your explicit agreement, like "I agree to change the license of this project from Apache 2.0 to MIT" (well you can just copy it hah) for this to happen, hopefully you dont mind it.
If you do - I can exclude the changes that _you_ own (the localization file) from this repository and proceed without you.

**Comment by vectorassembly** [comment]
I agree to change the license of this project from Apache 2.0 to MIT.

**Comment by SonicJDF** [comment]
I agree to agree

**Comment by LorenzoPapi** [comment]
I agree to agree to the agreement

**Comment by Vexatos** [comment]
Naturally, I agree to changing the license.

**Comment by 3TUSK** [comment]
I agree to change the license of this project from Apache 2.0 to MIT.

> When did I push these commits?

**Comment by necauqua** [comment]
Perfect, I'll make a commit soon and close this issue, thanks to all of you.

> When did I push these commits?

Well you are on the list of contributors, your PR does not exist anymore (because I made a new github account back in the day when username changes were not added and re-pushed my repo here), but the commit and merge are here - see 481d0c5. Holy  heck it's 5 years old actually :)
